src/ontology/mondo-edit.obo

```diff
@@ -618782,6 +618780,15 @@ is_a: MONDO:0018912 {source="PMID:25945066"} ! Cushing syndrome
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6475" xsd:anyURI
 
+[Term]
+id: MONDO:1060134
+name: atherosclerotic cardiovascular disease
+def: "Any cardiovascular disease resulting from atherosclerosis." [NCIT:C35771]
+xref: NCIT:C35771 {source="MONDO:equivalentTo", source="DOID:2348"}
```

Comment:
we do not need "DOID:2348" as a source since you are the one who asserted that this NCIT was equivalent to to this Mondo term (not DO)